### PR TITLE
Add release notes for 1.1.0

### PR DESCRIPTION
**Pull request by katinthehatsite**
Related to: pfHvTO-bo-p2

## Proposed Changes

This PR adds release notes for 1.1.0 release.

## Testing Instructions

* Review release notes and check if all prominent changes are included

## Pre-merge Checklist

- [ ] Have you checked for TypeScript, React or other console errors?

### CODE REVIEW

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -1,3 +1,11 @@
+1.1.0
+=====
+* Introduced a new feature to export and import sites
```
Comment (katinthehatsite) [inline RELEASE-NOTES.txt:3]: There are two things that I am not sure about for this one:

* should we add it to a separate section that says something like "Feature"?
* what PR should I link here? There are many of the PRs that  implemented this feature

@wojtekn what do you think?

Comment (katinthehatsite) [reply RELEASE-NOTES.txt:3]: Sidenote: I looked through other apps that we release but I did not really find any consistency there

File: RELEASE-NOTES.txt [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+* Translated the tooltip for the close button on the modal #454
+* Added support for Ukrainian language #452
+* Increased space limit to 250 MB #446
```
Comment (wojtekn) [inline RELEASE-NOTES.txt]:
```suggestion
* Increased demo site space limit to 250 MB #446
```

File: RELEASE-NOTES.txt (hunk 1 above)
Comment (wojtekn) [reply RELEASE-NOTES.txt:3]: It's okay to skip PRs in this case. I think it is good as is, or we could simplify it and say 'Added import and export features'.

**Review by wojtekn — APPROVED** [review]

File: RELEASE-NOTES.txt (hunk 1 above)
Comment (katinthehatsite) [reply RELEASE-NOTES.txt:3]: Sounds good, thank you!

File: RELEASE-NOTES.txt (hunk 2 above)
Comment (katinthehatsite) [reply RELEASE-NOTES.txt]: Updated in https://github.com/Automattic/studio/pull/472/commits/b881e0019a74b1d3c87f78462e1d900dafcd0ba8

**Pull request merged**